CMakeLists.txt

```diff
@@ -93,7 +93,8 @@ if (TEST)
   # ---- tests ----
   file(GLOB_RECURSE oadrtest_SRC ./oadrtest/oadrtest/*.cpp)
   add_executable(testoadr ${oadrtest_SRC})
-  target_link_libraries (testoadr LINK_PUBLIC oadr oadrsd curl xerces-c xml-security-c gtest pthread)
+  target_include_directories(testoadr PRIVATE oadrtest)
```

Comment:
I added this so the tests can include headers from oadrtest folder treating it as the starting point. I needed that to get rid of include paths referring to parent directories (e.g. "../../*.h").